test/validate/ferc1_test.py

```diff
@@ -160,3 +164,116 @@ def test_unique_rows_ferc1(pudl_out_ferc1, live_dbs, df_name, unique_subset):
         subset=unique_subset,
         df_name=df_name,
     )
+
+
+@pytest.mark.parametrize(
+    "table_name",
+    [
+        "core_ferc1__yearly_sales_by_rate_schedules_sched304",
+        "core_ferc1__yearly_pumped_storage_plants_sched408",
+        "core_ferc1__yearly_steam_plants_sched402",
+        "core_ferc1__yearly_hydroelectric_plants_sched406",
+        "core_ferc1__yearly_transmission_lines_sched422",
+    ],
+)
+def test_filter_for_freshest_data(
+    ferc1_engine_xbrl, table_name: pd.DataFrame
+) -> pd.DataFrame:
+    """Test if we are unexpectedly replacing records during filter_for_freshest_data.
+
+    NOTE: None of this actually calls any of the code over in pudl.transform.ferc1 rn.
+    That's maybe okay because we are mostly testing the data updates mostly here.
```

Comment:
does this feel okay? we could pull some of the bits from the transform filter out into smaller functions to actually run here... but i don't think it is worth it looking at the bits and the setup.

Reply:
hm after trying to do this, the main snag here is that `filter_for_freshest` does the prep to get the `never_duped` and `duped_groups`, does the filtering, then concats them back together.

So instead of passing in an optional `deduper` method I think it'll be simpler to pass in a `compare_methods` bool, which defaults to `False`